taprpc/taprootassets.proto

```diff
@@ -1018,6 +1018,10 @@ message SubscribeReceiveEventsRequest {
     // Filter receives by a specific address. Leave empty to get all receive
     // events for all addresses.
     string filter_addr = 1;
+
+    // The start time as a unix timestamp in seconds. If not set (default value
+    // 0), the daemon will start streaming events from the current time.
+    uint64 start_time = 2;
```

Comment:
Ah ok, it's in this PR. 

I do wonder if a monotonic event ID will be easier for devs to handle than a timestamp though.

Reply:
I personally think a timestamp is more useful, as it contains some information about the records by itself already. And since we're using a timestamp with microsecond precision, we effectively get monotonically increasing values as well.
Happy to add the ID as well though. But we'd need to also change the event subscription interface for the `deliverFrom` parameter, which in this case would need to be `fn.Either[time.Time, uint64]`.